app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -28,6 +28,10 @@ export default function NoteTable({
             </th>
             <th className="text-left">Title</th>
             <th className="text-left">Description</th>
+            <th className="text-center">Tags</th>
+            <th className="text-center">Created Date</th>
+            <th className="text-center">Due Date</th>
+            <th className="text-center">Conact</th>
```

Comment:
`Contact`
Also, please create an empty `th` which corresponds to the Icon Action Buttons in the table row.